static/app/utils/url/useSetQueryFieldInLocation.tsx

```diff
@@ -0,0 +1,19 @@
+import {useCallback} from 'react';
```

Comment:
intentional file addition?

Reply:
oops, that's from https://github.com/getsentry/sentry/pull/74096